src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoMonthGenitiveNames.cs

```diff
@@ -11,134 +12,52 @@ namespace System.Globalization.Tests
-        [Fact]
-        public void TestRuCulture()
-        {
-            DateTimeFormatInfo info = new CultureInfo("ru-RU").DateTimeFormat;
-
-            string[] expected = new string[] {
-                    "\u044F\u043D\u0432\u0430\u0440\u044F",
-                    "\u0444\u0435\u0432\u0440\u0430\u043B\u044F",
-                    "\u043C\u0430\u0440\u0442\u0430",
-                    "\u0430\u043F\u0440\u0435\u043B\u044F",
-                    "\u043C\u0430\u044F",
-                    "\u0438\u044E\u043D\u044F",
-                    "\u0438\u044E\u043B\u044F",
-                    "\u0430\u0432\u0433\u0443\u0441\u0442\u0430",
-                    "\u0441\u0435\u043D\u0442\u044F\u0431\u0440\u044F",
-                    "\u043E\u043A\u0442\u044F\u0431\u0440\u044F",
-                    "\u043D\u043E\u044F\u0431\u0440\u044F",
-                    "\u0434\u0435\u043A\u0430\u0431\u0440\u044F",
-                    ""
-                    };
-            VerificationHelper(info, expected, false);
-        }
-
-        // NegTest1: ArgumentNullException should be thrown when The property is being set to a null reference
-        [Fact]
-        public void TestNull()
+        public static IEnumerable<object[]> MonthGenitiveNames_TestData()
         {
-            Assert.Throws<ArgumentNullException>(() =>
+            yield return new object[] { DateTimeFormatInfo.InvariantInfo, new string[] { "January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December", "" } };
+            yield return new object[] { new CultureInfo("ru-RU").DateTimeFormat, new string[]
             {
-                new DateTimeFormatInfo().MonthGenitiveNames = null;
-            });
-            Assert.Throws<ArgumentNullException>(() =>
-            {
-                new DateTimeFormatInfo().MonthGenitiveNames = new string[] {
-                    "1",
-                    "2",
-                    "3",
-                    null,
-                    "5",
-                    "6",
-                    "7",
-                    "8",
-                    null,
-                    "10",
-                    "11",
-                    "12",
-                    ""
-                    };
-            });
+                "\u044F\u043D\u0432\u0430\u0440\u044F",
+                "\u0444\u0435\u0432\u0440\u0430\u043B\u044F",
+                "\u043C\u0430\u0440\u0442\u0430",
+                "\u0430\u043F\u0440\u0435\u043B\u044F",
+                "\u043C\u0430\u044F",
+                "\u0438\u044E\u043D\u044F",
+                "\u0438\u044E\u043B\u044F",
+                "\u0430\u0432\u0433\u0443\u0441\u0442\u0430",
+                "\u0441\u0435\u043D\u0442\u044F\u0431\u0440\u044F",
+                "\u043E\u043A\u0442\u044F\u0431\u0440\u044F",
+                "\u043D\u043E\u044F\u0431\u0440\u044F",
+                "\u0434\u0435\u043A\u0430\u0431\u0440\u044F",
+                ""
+            } };
```

Comment:
Style nit-pref.   I don't think we have a declarative preferred style here, but I like chopping before the first value, and that keeps indentation levels scoped:

```C#
yield return new object[]
{
    new CultureInfo("ru-RU").DateTimeFormat,
    new string[]
    {
        "\u044F\u043D\u0432\u0430\u0440\u044F",
        ...
    },
};

That also eliminates things like the double-close-curly.

For me it just helps my eyes focus on where the different pieces come from/go.